dqgen/resources/html_templates/static/macros.html

```diff
@@ -58,4 +58,15 @@
     {% else %}
         {{ render_error("How did you get here? did you forget to use 'render_fetch_results' macro?") }}
     {% endif %}
-{%- endmacro %}
\ No newline at end of file
+{%- endmacro %}
+
+{% macro count_value(df) %}
```

Comment:
does this mean that we do not need statistics queries and we simply count rows of the normal queries?